src/Marten/Linq/CollectionAnyContainmentWhereFragment.cs

```diff
@@ -161,7 +154,17 @@ private string buildSubQuery(SubQueryExpression subQuery, CommandBuilder command
             {
                 throwNotSupportedContains();
             }
-            var fromParam = command.AddParameter(from.Value);
+
+            var values = new List<string>();
```

Comment:
Why are we having to do this? Is this something that got broken by npgsql 4?

Reply:
It seems so, we're passing the parameter as array of text (see [here](https://github.com/JasperFx/marten/pull/1045/files/e56226f5ff844a03a847ca0582459f0812161efd#diff-f98a5f4de4c572b88fea044c7ef852a8R168) ). I believe that previously it was the automated conversion of parameter, but right now it's not being performed. It throws exception that TextHandler cannot handle array of guids (or something like that). In general it's because Array of String != Array of Guids.